pom.xml

```
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.399</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>2.37</version><!-- which version of Jenkins is this plugin built against? -->
```

I don't quite remember correctly, but I think this change was needed to get the build to pass.

---

```suggestion
    <version>4.12</version>
```

For the latest stuff. 2.37 is pretty old